CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ When submitting a pull request, please make sure to:
 
 To set up a development environment for this project, you will need to:
 
-* Install Python 3.9 or higher and make sure java is installed on your system
+* Install Python 3.11 or higher and make sure java is installed on your system
```

Comment:
I believe java is required for one dataset, `CAISO().get_curtailment()`, which uses `tabula-py`. 

https://github.com/gridstatus/gridstatus/blob/a92f1d10ec2a768d130b9235779df085f1047541/gridstatus/caiso.py#L878

https://pypi.org/project/tabula-py/

It would be great to remove this dependency.